From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Fri, 5 Jun 2026 17:15:17 +0800
Subject: submitted-dl-support

older versions of glibc would build dl-sysdep as shared-only and dl-support as
static-only.  alpha hooks in a cache variable via dl-auxv.h.  newer versions of
glibc build dl-sysdep as both shared and static which means we now have symbol
duplication for static builds with dl-sysdep and dl-support.  since dl-sysdep
is both shared/static, there is no point in hooking dl-support anymore, so we
can punt it.
---
 sysdeps/unix/sysv/linux/alpha/dl-support.c | 2 --
 1 file changed, 2 deletions(-)
 delete mode 100644 sysdeps/unix/sysv/linux/alpha/dl-support.c

diff --git a/sysdeps/unix/sysv/linux/alpha/dl-support.c b/sysdeps/unix/sysv/linux/alpha/dl-support.c
deleted file mode 100644
index 2902176..0000000
--- a/sysdeps/unix/sysv/linux/alpha/dl-support.c
+++ /dev/null
@@ -1,2 +0,0 @@
-#include "dl-auxv.h"
-#include <elf/dl-support.c>
